backend/.env.example

```
@@ -1,24 +1,16 @@
-# Server
-PORT=3000
+# Server Configuration
+PORT=5000
+MONGO_URI=mongodb+srv://<username>:<password>@cluster0.mongodb.net/vigybag
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The MONGO_URI key should go before the PORT key

(UnorderedKey)